### DISCUSSION TITLE
[Question] How to open file encrypted with SQLitePCLRaw.bundle_e_sqlcipher (.Net)?

### DISCUSSION THREAD

**Discussion by BieleckiLtd** [pawelsalawa/sqlitestudio#3878] [Q&A]
SQLitePCLRaw.bundle_e_sqlcipher nuget package is the default 'go to' when encrypting database with C# and .Net (https://docs.microsoft.com/en-us/dotnet/standard/data/sqlite/encryption?tabs=netcore-cli). I can't seem to find a way to open file in SQLiteStudio encrypted this way. Any ideas? Thanks.

**Comment by pawelsalawa** [comment]
SQLCipher (which you're referring to) is supported by the https://github.com/pawelsalawa/sqlitestudio/wiki/DbSqliteCipher plugin. The problem is that it's a little bit outdated (update is pending) and it may have issues opening databases created with recent SQLCipher engine.
